src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Framework.FileSystemGlobbing.Abstractions
```

Comment:
`FileProviderGlobbing` instead as we changed `FileSystem` to `FileProvider` everywhere?